python/pfs/instmodel/makeSim.py

```diff
@@ -84,7 +90,25 @@ def makeSim(detector, pfsDesignId=0, fiberFilter=None,
                          for ii in range(len(design))]
     else:
         doSkyForFiber = np.zeros_like(fibers, dtype=bool)
-    source = LightSource(domeOpen, lamps, skyModel, design, spectraDir)
+
+    if objSpectraDir:
+        catConfigPath = os.path.join(objSpectraDir, catConfig)
+        with open(catConfigPath, 'r') as stream:
+            catConfigData = yaml.safe_load(stream)
+        # Convert yaml data to a dict indexed by catId,
```

Comment:
You haven't provided any documentation about the expected format of this YAML file, nor provided an example or suitable default.